docs/reference/setup/install/windows.asciidoc

```diff
@@ -76,12 +76,12 @@ TIP: Ensure the installation machine has access to the internet and that any cor
 [[msi-installer-selected-plugins]]
 image::images/msi_installer/msi_installer_selected_plugins.png[]
 
-As of version 6.3.0, {xpack} is now https://www.elastic.co/products/x-pack/open[bundled by default]. The final step allows a choice of the type of {xpack} license to install, in addition to security configuration and built-in user configuration:
+As of version 6.3.0, X-Pack is now https://www.elastic.co/products/x-pack/open[bundled by default]. The final step allows a choice of the type of X-Pack license to install, in addition to security configuration and built-in user configuration:
 
 [[msi-installer-xpack]]
 image::images/msi_installer/msi_installer_xpack.png[]
 
-NOTE: {xpack} includes a choice of a Trial or Basic license for 30 days. After that, you can obtain one of the
+NOTE: X-Pack includes a choice of a Trial license (valid for 30 days) or Basic license (valid perpetually). After that, you can obtain one of the
```

Comment:
Might be a bit clearer to separate out the license types from their validity e.g.

X-Pack includes a choice of a Trial or Basic license. A Trial license is valid for 30 days, after which you can obtain one of  the available subscriptions. The Basic license on the other hand is free and perpetual. Consult the https://www.elastic.co/subscriptions[available subscriptions] for further details on which features are available under which license.